src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -583,8 +583,8 @@ public void addPayloadType(PayloadType payloadType)
     public long getLastActivity()
     {
         PacketIOActivity packetIOActivity
-                = this.dtlsTransport.getPacketIOActivity();
-        return packetIOActivity.getLastOverallActivityTimestampMs();
+                = this.transceiver.getPacketIOActivity();
+        return packetIOActivity.getLatestOverallActivity().getEpochSecond();
```

Comment:
this was millis before but you've got it as seconds here

Reply:
Nice catch, thanks Brian! Now fixed.